Cargo.toml

```diff
@@ -31,6 +31,7 @@ lazy_static = "1"
 pulldown-cmark = "0.7"
 itertools = "0.9"
 crossterm = "0.17"
+terminal_size = "0.1.12"
```

Comment:
I would prefer not to introduce another dependency, `crossterm` can has a `fn size()` which does the same :) https://docs.rs/crossterm/0.17.5/crossterm/terminal/fn.size.html

Reply:
true, I missed it. Done here: https://github.com/drahnr/cargo-spellcheck/pull/56/commits/b10108188ab418c44956d554666a7fab215e6ed1

Reply:
Can you also remove it from `Cargo.toml` :) thanks!

Reply:
Removed, it was also in .lock but did not clean up automatically for some reason. isn't weird or maybe I misuderstood.

`Cargo.lock `
done: https://github.com/drahnr/cargo-spellcheck/pull/56/commits/c65c12516bcc4ae0002804daff37870a37b7224c

`Cargo.toml`
done: https://github.com/drahnr/cargo-spellcheck/pull/56/commits/45a619084d692bf2164e131f977eaf5b191b2934